### DISCUSSION TITLE
The problems on Delaunay network

### DISCUSSION THREAD

**Discussion by Devin-one** [PMEAL/OpenPNM#2409] [Q&A] [answered]
Hi, Professor. 
I'm studying according to the tutorial recently，https://github.com/PMEAL/OpenPNM/blob/dev/examples/tutorials/network/adding_boundary_pores.ipynb . But when I run according to the example, the code reports an error. I don't know what the problem is.
![image](https://user-images.githubusercontent.com/93040662/166698625-87ad0466-3bb1-4876-9073-3af63f209a60.png)
By the way, I want to design  a 2D square domain with a length of 300 and a width of 200. Can I change the shape parameters to realize my idea? Because I changed the shape parameter in an example of creating a cubic and found that  the values of dimension x, y and z was 1 by default.
![image](https://user-images.githubusercontent.com/93040662/166715891-c1fbc050-5006-4bbc-83ca-0427133b07d0.png)
Both of these questions puzzled me very much. Sorry to bother you, but I’d appreciate some help.

**Comment by Xiaozizhuo** [comment, marked as answer]
This problem is easy to solve, open the definition of Delaunay function, you will find that point should be changed to num_point

**Reply by Devin-one** [reply]
Success! Thank you very much. Yet, I else have some questions to ask you. Can I add your contact information? If  you don't mind, can you send your contact information directly to my personal email.

**Reply by jgostick** [reply]
Great to hear!  Yes, contact me at jgositck at gmail.

**Reply by jgostick** [reply]
Unless you were referring to @Xiaozizhuo ?  I always just assume people are using this forum to ping the dev directly.
